app/pages/shows/index.vue

```diff
@@ -74,6 +75,9 @@ export default {
     }
   },
   computed: {
+    ...mapGetters({
+      fullSchedule: 'returnArcsiShows'
```

Comment:
Achtung: `fullSchedule` is probably a typo here as the `/shows` page does not contain schedule info. The functionality is looking good though.

Reply:
Btw, the arcsi `schedule` API is too general at the moment (it returns all shows w/o the items -> it should be called schedule).

Reply:
I'm not sure about this, if this is the `/shows` page I would recommend to use the `/show/list` API route here.
You are right Arcsi's schedule API returns every _active_ show w/o the items, maybe I could change it later to be more specific

Reply:
Yeah, it uses the /all_without_items API via app/constants/index.js, so it's only the naming wrong here.